### PR TITLE
Update cut_videos.sh

### PR DESCRIPTION
**Pull request by kennymckormick**
Remove frame rate adjusting in cut_videos.sh

### CODE REVIEW

**Comment by irvingzhang0512** [comment]
I'm not familar with `build_rawframes.py`. Can we use opencv to extract rgb/flow **30fps** frames?

**Comment by kennymckormick** [comment]
> I'm not familar with `build_rawframes.py`. Can we use opencv to extract rgb/flow **30fps** frames?

Extracting frames with Opencv is also OK, just need to make sure the setting for extraction (like 30fps) is correct.

**Comment by irvingzhang0512** [comment]
Please correct me if I miss anything.

1. Run `bash cut_videos.sh` without `-r 30` to cut videos
2. Run `bash extract_rgb_frames_opencv.sh` to extract rgb frames. Now we are expecting that the number of frames for each video is 27030.

However, I got results like

```shell
Total number of videos found:  299
rgb 70 c9pEMjPT16M.webm None done
build_rawframes.py:60: UserWarning: Length inconsistent!Early stop with 22526 out of 22526 frames.
  'Length inconsistent!'
rgb 20 KWoSGtglCms.mkv None done
build_rawframes.py:60: UserWarning: Length inconsistent!Early stop with 27004 out of 27004 frames.
  'Length inconsistent!'
rgb 40 HVAmkvLrthQ.mkv None done
build_rawframes.py:60: UserWarning: Length inconsistent!Early stop with 27004 out of 27004 frames.
  'Length inconsistent!'
rgb 60 fpprSy6AzKk.mkv None done
build_rawframes.py:60: UserWarning: Length inconsistent!Early stop with 27031 out of 27031 frames.
  'Length inconsistent!'
rgb 0 yvgCGJ6vfkY.mkv None done
build_rawframes.py:60: UserWarning: Length inconsistent!Early stop with 27004 out of 27004 frames.
  'Length inconsistent!'
rgb 30 fD6VkIRlIRI.mkv None done
build_rawframes.py:60: UserWarning: Length inconsistent!Early stop with 27004 out of 27004 frames.
  'Length inconsistent!'
```

File: tools/data/ava/cut_videos.sh [hunk 1]
```diff
@@ -29,6 +29,6 @@ for video in $(ls -A1 -U ${IN_DATA_DIR}/*)
 do
   out_name="${OUT_DATA_DIR}/${video##*/}"
   if [ ! -f "${out_name}" ]; then
-    ffmpeg -ss 900 -t 901 -r 30 -i "${video}" -strict experimental "${out_name}"
+    ffmpeg -ss 900 -t 901 -i "${video}" -strict experimental "${out_name}"
```
Comment (innerlee) [inline tools/data/ava/cut_videos.sh:32]: try to move `-r 30` after `-i video`, as discussed here https://github.com/open-mmlab/mmaction2/issues/534#issuecomment-770386690

**Review by innerlee — APPROVED** [review]

**Pull request merged**